subprojects/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/processor/AbstractInstrumentationProcessor.java

```diff
@@ -60,11 +62,13 @@
 @SupportedSourceVersion(SourceVersion.RELEASE_8)
 public abstract class AbstractInstrumentationProcessor extends AbstractProcessor {
 
+    public static final String PROJECT_NAME_OPTIONS = "org.gradle.annotation.processing.instrumented.project";
+
     protected abstract Collection<InstrumentationProcessorExtension> getExtensions();
 
     @Override
     public Set<String> getSupportedOptions() {
-        return Collections.singleton("org.gradle.annotation.processing.aggregating");
+        return new HashSet<>(Arrays.asList("org.gradle.annotation.processing.aggregating", PROJECT_NAME_OPTIONS));
```

Comment:
💅 Any reason not to use `ImmutableSet.copyOf()` here?

Reply:
It's just because we don't have a dependency on guava in ann. processor